examples/aggregator/client/aggregator.cc

```diff
@@ -79,7 +79,7 @@ int main(int argc, char** argv) {
   // This particular value is taken from
   // https://github.com/project-oak/oak/blob/abb0bbff11b60f90be4df2cc4446be58b98015ff/reproducibility_index#L3
```

Comment:
This comment seems to be outdated by the change below &ndash; could we either update it in sync or replace the comment with something more generic, e.g.
```Rust
  // The particular value corresponds to the hash on the `aggregator.wasm` line
  // in https://github.com/project-oak/oak/blob/hashes/reproducibility_index
```

Reply:
Good point, thanks! FWIW I think the UX of upgrading modules if far from optimal, we definitely need more work on it.